UI/frontend-plugins/frontend-tools/data/scripts/instant-replay.lua

```diff
@@ -2,6 +2,9 @@ obs         = obslua
 source_name = ""
 hotkey_id   = obs.OBS_INVALID_HOTKEY_ID
 attempts    = 0
+last_replay = ""
+max_attempts= 100
```

Comment:
I'd leave `max_attempts` at 10 to avoid changing behavior from the original script.